paasta_tools/cli/cmds/logs.py

```diff
@@ -121,18 +152,31 @@ def completer_clusters(prefix, parsed_args, **kwargs):
         return list_clusters()
 
 
+def color_component(component, color_param):
+    # If the color field is iterable try to join
+    # all the colors together, otherwise just apply
+    # the single color
+    try:
+        for color in color_param:
```

Comment:
Please see this [line](https://github.com/Yelp/paasta/pull/559/files#diff-d3b2beac0ebf41c3a27cbe71e22e935aR450) right here.

It's mostly so I can get this combination of yellow and bold in the help output which looks pretty nice I think:
![paasta logs help](https://cloud.githubusercontent.com/assets/773529/15829592/3ff67f86-2bc9-11e6-8011-1bae42888997.png)